src/controllers/user.controller.ts

```diff
@@ -31,7 +32,8 @@ export const getAllUsers: RequestHandler = async (
     );
 
     if (suggestedUser) {
-      return res.json(suggestedUser);
+      res.json(suggestedUser);
+      return;
```

Comment:
let's use next() after res.json(suggestedUser) in other case another middleware never get control

Reply:
added next (), thanks for explanations